draft-ietf-masque-connect-udp.md

```diff
@@ -403,18 +428,89 @@ Reference:
 : This document.
 
 
-## Datagram Format Type {#iana-format-type}
+--- back
 
-This document will request IANA to register UDP_PAYLOAD in the "HTTP Datagram
-Format Types" registry established by {{HTTP-DGRAM}}.
+# Example Extensions
```

Comment:
I feel that these extensions are a little too well formed to serve well as extensions.  They read far too much like specifications, which might be misleading or confusing.

If the goal is to guide future protocol designers, that guidance can be provided as pure text.  Including "grammar" (illustrations really, if I stick to my position there) isn't necessary to convey the necessary meaning.

Reply:
I don't disagree, but since we're most likely to remove this appendix entirely before publication, I'd rather not spend too much time on it. I've added Example- in more places to make this clearer